dvc/command/repro.py

```diff
@@ -92,7 +92,7 @@ def add_arguments(repro_parser):
         "--pipeline",
         action="store_true",
         default=False,
-        help="Reproduce the whole pipeline that the specified stage file "
+        help="Reproduce the whole pipeline that the specified targets "
         "belongs to.",
```

Comment:
```suggestion
        "belong to.",
```

Reply:
Thanks. Fixed

Reply:
[offtopic] @pmrowla should I click **Resolve conversation** button here when I fixed a reviewer's note? Or the reviewer should do it himself?

Reply:
For something minor like this it's fine to mark it resolved yourself, but if you're unsure about something it's also fine to leave it for the reviewer to decide